### PR TITLE
Fix: Unparam linter failures

### PR DESCRIPTION
**Pull request by sibicramesh**
--> Fix linters

### CODE REVIEW

**Comment by sibicramesh** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix-linters",
    "component": "enforcerd",
    "pr-id": "992",
    "commit-sha": "f50c57c29397d25f7418257791eb54c9dca6c072"
  },
  {
    "project": "fix-linters",
    "component": "trireme-lib",
    "pr-id": "718",
    "commit-sha": "61e262d53f2208d0272117f0e7c8c04f3167282f"
  }
]
```

**Comment by sibicramesh** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "enforcerd-reporting",
    "component": "enforcerd",
    "pr-id": "993",
    "commit-sha": "d74ed8826868c64d49ea3c19b9a897e4f83b7918"
  },
  {
    "project": "enforcerd-reporting",
    "component": "trireme-lib",
    "pr-id": "718",
    "commit-sha": "ab0a99106ab5f219c912f55b9d497f02660962e0"
  }
]
```

File: controller/pkg/pkiverifier/pkiverifier.go [hunk 1]
```diff
@@ -83,7 +83,7 @@ func (p *tokenManager) Verify(token []byte) (*ecdsa.PublicKey, error) {
 			continue
 		}
 
-		JWTToken, err = jwt.ParseWithClaims(tokenString, claims, func(token *jwt.Token) (interface{}, error) {
+		JWTToken, err = jwt.ParseWithClaims(tokenString, claims, func(_ *jwt.Token) (interface{}, error) { // nolint
```
Comment (sibicramesh) [inline controller/pkg/pkiverifier/pkiverifier.go:86]: // nolint is required because of return parameter being nil always

**Comment by sibicramesh** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "enforcerd-reporting",
    "component": "enforcerd",
    "pr-id": "993",
    "commit-sha": "d74ed8826868c64d49ea3c19b9a897e4f83b7918"
  },
  {
    "project": "enforcerd-reporting",
    "component": "trireme-lib",
    "pr-id": "718",
    "commit-sha": "3efc44910f85f39b9cb8c9bcc77eb4049d88e7ce"
  }
]
```

File: controller/internal/supervisor/iptablesctrl/acls_test.go [hunk 2]
```diff
@@ -934,15 +934,15 @@ func TestAddAppACLs(t *testing.T) {
 				},
 			}
 
-			ipsets.MockNewIpset(t, func(name string, hasht string, p *ipset.Params) (provider.Ipset, error) {
+			ipsets.MockNewIpset(t, func(_ string, _ string, _ *ipset.Params) (provider.Ipset, error) { // nolint
```
Comment (satyamsi) [inline controller/internal/supervisor/iptablesctrl/acls_test.go]: why do we still need // nolint ?

File: controller/internal/supervisor/iptablesctrl/acls_test.go [hunk 3]
```diff
@@ -1030,15 +1030,15 @@ func TestAddNetACLs(t *testing.T) {
...
```
Comment (satyamsi) [inline controller/internal/supervisor/iptablesctrl/acls_test.go]: why do we still need //nolint ?

**Review by satyamsi — APPROVED** [review]

**Pull request merged**